docs/setup/native.md

```diff
@@ -60,25 +60,11 @@ the `vets-api` directory, run `./bin/setup-osx && source ~/.bash_profile && cd
    - More information about installing *with* Sidekiq Enterprise as well as our credentials are on the internal system here: https://github.com/department-of-veterans-affairs/vets-api#authentication-required-for-enterprisecontribsyscom
 
 11. Install overcommit `overcommit --install --sign`
-12. Setup localhost certificates / keys:
-   - Create certs directory within config:  `mkdir ./config/certs`
-   - Copy the [certificate][certificate] to `./config/certs/vetsgov-localhost.crt`
-   - Copy the [key][key] to `./config/certs/vetsgov-localhost.key`
-   - *NOTE*: If you don't have access to these keys, running the following
-     commands will provide basic functionality, such as for running unit tests:
-   - `touch ./config/certs/vetsgov-localhost.crt`
-   - `touch ./config/certs/vetsgov-localhost.key`
-   
-   - *NOTE:* using `touch` to create blank cert and key files means that local authentication with IDme will not work
-   
-   [certificate]: https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Identity/Login/IDme/development-certificates/vetsgov-localhost.crt
-   [key]: https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Identity/Login/IDme/development-certificates/vetsgov-localhost.key
-
```

Comment:
All I did was remove section 12. 

Github is showing a strange diff because I also had to re-number the bullet points after number 12.